test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -1023,19 +1018,6 @@ describe('TwilioConnectionTransport', () => {
         it('should not emit either "connected" or "message" events', () => {
           assert(!connectedOrMessageEventEmitted);
         });
-
-        it('should call .sendMessage on the underlying TwilioConnection with a Connect RSP message', () => {
-          const message = test.twilioConnection.sendMessage.args[0][0];
-          assert.equal(message.name, test.name);
-          assert.equal(message.participant, test.localParticipantState);
-          assert.deepEqual(message.peer_connections, test.peerConnectionManager.getStates());
-          assert.equal(message.token, test.accessToken);
-          assert.equal(message.type, 'connect');
-          assert.equal(message.version, 2);
-          assert.equal(message.publisher.name, `${name}.js`);
-          assert.equal(message.publisher.sdk_version, version);
-          assert.equal(typeof message.publisher.user_agent, 'string');
-        });
```

Comment:
It may still be worth it to test sending an update sent while connecting after the hello body.

Reply:
@ceaglest We [enqueue](https://github.com/twilio/twilio-video.js/blob/master/lib/signaling/v2/twilioconnectiontransport.js#L291) all `update` messages until we receive "connected" message from Room Service.